src/PartyDropper.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import "openzeppelin-contracts/token/ERC1155/ERC1155.sol";
+import "openzeppelin-contracts/token/ERC1155/extensions/ERC1155Supply.sol";
+import "openzeppelin-contracts/access/Ownable.sol";
+import "./Base64.sol";
+import "./IPartyBid.sol";
+
+contract PartyDropper is ERC1155Supply, Ownable {
+    struct Edition {
+        IPartyBid party;
+        string name;
+        string imageURI;
+        string description;
+    }
+    mapping(uint256 => Edition) public editions;
+    mapping(uint256 => mapping(address => bool)) public editionMinter;
+    mapping(uint256 => address) public editionCreator;
+    mapping(address => bool) public allowedCreators;
+    uint256 lastEditionId;
+
+    event EditionCreated(
+        uint256 indexed editionId,
+        address indexed creator,
+        address party,
+        string name
+    );
+    event Claimed(address indexed user, uint256 editionId);
+
+    modifier onlyAllowedCreator() {
+        require(
+            allowedCreators[msg.sender] || msg.sender == owner(),
+            "must be allowed creator or owner"
+        );
+        _;
+    }
+
+    constructor() ERC1155("") {}
+
+    // public functions
+    function mintEdition(uint256 editionId) public {
```

Comment:
Maybe call this mintFromEdition, so its not confused with creating a new edition?  I'm less familiar with the standard terminology for 1155s so def nabd

Reply:
this is a good idea 👍